src/codecoverage/bot/code_coverage_bot/phabricator.py

```diff
@@ -118,12 +117,23 @@ def generate(self, report, changesets=None):
                         # This means the file has been removed by this changeset, and maybe was brought back by a following changeset.
                         continue
 
+                    # Calc lines added by this patch
+                    lines_added = sum([
+                        parse_revision_id(line['desc']) == revision_id
```

Comment:
You should be able to do this by checking the changeset of the line rather than parsing the description.
If the changeset is the changeset of interest, then it's a line added by this patch.